contracts/src/v0.8/feeds/DualAggregator.sol

```diff
@@ -56,6 +56,9 @@ contract DualAggregator is OCR2Abstract, Ownable2StepMsgSender, AggregatorV2V3In
 
   bytes32 internal s_latestConfigDigest;
 
+  // Overhead incurred by accounting logic.
```

Comment:
Removing this from the HotVars struct means we're net even on storage slots. I think something you can investigate would be to add more state variables into HotVars to maximize the 2 slots. 

For example we've added the following:
secondaryProxy = 20 bytes 
s_cutoffTime = 4 bytes
s_maxSyncIterations = 4 bytes
s_latestSecondary = 1 byte
latestSecondaryRoundId = 4 bytes

There's a lot of packing opportunities here.

Reply:
To further investigate if we can reduce the size of cutoffTime and maxSyncIterations

Reply:
i_secondaryProxy is not part of the hot path so ignore that one.